data/gaTable.yaml

```diff
@@ -567,7 +567,7 @@
   analyticsContact: 'David Watkins'
   analyticsContactEmail: 'wwatkins@usgs.gov'
 
-- longName: Grand Canyon Monitoring Research Center Website
+- longName: Grand Canyon Monitoring and Research Center''s Discharge, Sediment, and Water Quality Monitoring App
```

Comment:
i just tried out `y <- yaml::yaml.load_file('data/gaTable.yaml')`. with just one `'`, no escaping, it looks fine to me:
```

- longName: Grand Canyon Monitoring and Research Center's Discharge, Sediment, and Water Quality Monitoring App
  shortName: GCMRC
  accountName: CIDA GCMRC Analytics
  viewID: '80089136'
```

```
y <- yaml::yaml.load_file('data/gaTable.yaml')
y[[27]]$longName
## [1] "Grand Canyon Monitoring and Research Center's Discharge, Sediment, and Water Quality Monitoring App"
```